setup.py

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env python
 
 from setuptools import setup, find_packages
+
 import provider
 
 setup(
-    name='django-oauth2-provider',
+    name='edx-django-oauth2-provider',
```

Comment:
Ok.  It may be nice to know what the error is in case it isn't uninstalled, so you can message edx-code accordingly.